site/docs/relations/logical_relations.md

```diff
@@ -268,14 +268,23 @@ The set operation encompasses several set-level operations that support combinin
+* m: the number of time a records occurs in the primary input
+* n1: the number of times a record occurs in the 1st secondary input
+* n2: the number of times a record occurs in the 2nd secondary input
+* ...
+* n: the number of times a record occurs in the nth secondary input
+
+| Operation                  | Description                                                                                                                                                                                                                                                   | Examples                                                                                                                                                         | Output Nullability
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------------------------
+| Minus (Primary)            | Returns all records from the primary input excluding any matching rows from secondary inputs, removing duplicate rows within or across sets.<br/>Each value is treated as a unique member of the set, so duplicates in the first set don’t affect the result. | MINUS<br/>&nbsp;&nbsp;m:  {1, 2, 2, 3, 3, 3, 4}<br/>&nbsp;&nbsp;n1: {1, 2}<br/>&nbsp;&nbsp;n2: {3}<br/>YIELDS<br/>{4}                                            | The same as the primary input.                                                                                                                                                                     
+| Minus (Primary All)        | Returns all records from the primary input excluding any matching records from secondary inputs.<br/>For each specific record returned, the output contains max(0, m - sum(n1, n2, …, n)) copies.                                                             | MINUS ALL<br/>&nbsp;&nbsp;m:  {1, 2, 2, 3, 3, 3, 3}<br/>&nbsp;&nbsp;n1: {1, 2, 3, 4}<br/>&nbsp;&nbsp;n2: {3}<br/>YIELDS<br/>{2, 3, 3}                            | The same as the primary input.
+| Minus (Multiset)           | Returns all records from the primary input excluding any records that are included in *all* secondary inputs.                                                                                                                                                 | MINUS MULTISET<br/>&nbsp;&nbsp;m:  {1, 2, 3, 4}<br/>&nbsp;&nbsp;n1: {1, 2}<br/>&nbsp;&nbsp;n2: {1, 2, 3}<br/>YIELDS<br/>{4}                                      | The same as the primary input.
```

Comment:
I think the example here is wrong?
```
MINUS MULTISET
  m: {1, 2, 3, 4}
  n1: {1, 2}
  n2: {1, 2, 3}
YIELDS
{4}
```
The output should be `{3, 4} because 3 is only included in n2 and not n1.

Reply:
Yes, thanks :)